Sources/DataConnectError.swift

```diff
@@ -1,4 +1,4 @@
-// Copyright 2024 Google LLC
+// Copyright 2025 Google LLC
```

Comment:
It's ok to change, but doesn't _need_ to. The copyright year when the file was created is sufficient. cc: @paulb777

Reply:
I see. I'll keep that in mind for next time.